File: cryptocurrencyWeb/src/main/java/com/outsourcebooster/cryptocurrency/web/model/NotificationRule.java
package com.outsourcebooster.cryptocurrency.web.model;

/**
 * Created by rklimemnko on 02.07.2016.
 */
public class NotificationRule {

    private Currency currency;
    private String price;

    public Currency getCurrency() {
        return currency;
    }

    public void setCurrency(Currency currency) {
        this.currency = currency;
    }

    public String getPrice() {
        return price;
    }

    public void setPrice(String price) {
        this.price = price;
    }
}

File: cryptocurrencyWeb/src/main/java/com/outsourcebooster/cryptocurrency/web/model/user/NotificationRule.kt
package com.outsourcebooster.cryptocurrency.web.model.user

import com.outsourcebooster.cryptocurrency.web.model.currency.CryptoCurrency

/**
 * Created by rklimemnko on 02.07.2016.
 */
data class NotificationRule(val currency: CryptoCurrency?,
                            val price: String) {
    constructor(): this(null, "")
}